### PR TITLE
Add Signal Smoke Grenades / FlagMarkers / Update Equipment

### PR DESCRIPTION
**Pull request by Lazejun**
## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [x] Enhancement

### What have you changed and why?
Information:
Added SignalSmokeGrenades to Templates and gave them to Squadleader,
Removed Colored smoke from Normal Smoke grenades and move them to SignalSmokeGrenades.

Added new AFRF Gear from #1882 to Arid, slight changes to the Temperate Template.

Added FlagMarkerType to Rebell Factions, this was already done for VN, but not for the other Factions.
Changed FIA and Napas Flagtexture to the FIA Flag.

Selector:
Changed Log Message to Template instead of Templates,
Added a Case for Worldname Altis in Vanilla to use the AAF Template as Occupants.

AAF Template:
Added a SniperRifle for the Militia Sniper, this was missed and would have left him with just his Pistol.

Nato Arid Template:
Changed one wrong Scope to the Normal Variant,
Changed one Pistol to the Tan Variant.

### Please specify which Issue this PR Resolves.
closes #XXXX

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
Play against AFRF and look at their Shiny new Gear,
As SignalSmokes are only given to SLs you could spawn some and look in their Inventory:
```sqf
//Some OCC Militia Units
[group player, "loadouts_occ_military_SquadLeader", getpos (player)] call A3A_fnc_createUnit;
[group player, "loadouts_occ_military_Rifleman", getpos (player)] call A3A_fnc_createUnit;
[group player, "loadouts_occ_military_Medic", getpos (player)] call A3A_fnc_createUnit;
[group player, "loadouts_occ_military_Engineer", getpos (player)] call A3A_fnc_createUnit;

//This is for your Own Safety
group player setCombatMode "BLUE";
group player setBehaviourStrong "CARELESS";
{
_x disableAI "TARGET"; 
_x disableAI "AUTOTARGET";
_x setspeaker "novoice";
_x stop true;
_x setCaptive true;
} forEach units group player;
```
********************************************************
Notes:

### CODE REVIEW

File: A3-Antistasi/Templates/NewTemplates/RHS/RHS_AI_AFRF_Arid.sqf [hunk 1]
```diff
@@ -313,26 +310,28 @@ _militaryLoadoutData setVariable ["sniperRifles", [
...
+_policeLoadoutData setVariable ["vests", ["rhs_6b2_lifchik", "rhs_6b2_chicom"]];
+_policeLoadoutData setVariable ["helmets", ["rhs_cossack_visor_cap", "rhs_cossack_papakha"]];
 _policeLoadoutData setVariable ["rifles", [
-["rhs_weap_ak103_1", "rhs_acc_dtk", "", "", ["rhs_10Rnd_762x39mm"], [], ""]
-]];			//this line determines the rifles for police loadouts -- Example: ["arifle_MX_F", "arifle_MX_pointer_F"] -- Array, can contain multiple assets
-_policeLoadoutData setVariable ["smgs", ["rhs_weap_pp2000"]];			//this line determines the smgs for police loadouts -- Example: ["SMG_01_F", "SMG_01_Holo_F"] -- Array, can contain multiple assets
-_policeLoadoutData setVariable ["antiInfantryGrenades", ["rhs_mag_fakel", "rhs_mag_fakels", "rhs_mag_plamyam"]];
+["rhs_weap_akmn", "rhs_acc_dtk", "", "", ["rhs_10Rnd_762x39mm"], [], ""]
+]];
+_policeLoadoutData setVariable ["shotgun", [
```
Comment (jaj22) [inline A3-Antistasi/Templates/NewTemplates/RHS/RHS_AI_AFRF_Arid.sqf:319]: Should probably be `shotguns`, for consistency. All the other vars are plural.

File: A3-Antistasi/Templates/NewTemplates/RHS/RHS_AI_AFRF_Arid.sqf [hunk 2]
```diff
@@ -254,15 +255,11 @@ _sfLoadoutData setVariable ["sidearms", [
...
-_militaryLoadoutData setVariable ["SLvests", ["rhs_6b23_digi_6sh92_headset_mapcase", "rhs_6b23_digi_6sh92_headset"]];
-_militaryLoadoutData setVariable ["GLvests", ["rhs_6b23_digi_6sh92_vog"]];
-_militaryLoadoutData setVariable ["backpacks", ["rhs_assault_umbts", "B_Kitbag_sgg", "rhs_sidor"]];
-_militaryLoadoutData setVariable ["helmets", [
-"rhs_6b27m_digi", "rhs_6b27m_digi_bala", "rhs_6b27m_digi_ess", "rhs_6b27m_digi_ess_bala",
-"rhs_6b7_1m_emr", "rhs_6b7_1m_bala1_emr", "rhs_6b7_1m_bala2_emr", "rhs_6b7_1m_emr_ess", "rhs_6b7_1m_emr_ess_bala"
-]];
+_militaryLoadoutData setVariable ["vests", ["rhs_6b45_rifleman", "rhs_6b45_rifleman_2"]];
+_militaryLoadoutData setVariable ["SLvests", ["rhs_6b45_off"]];
+_militaryLoadoutData setVariable ["GLvests", ["rhs_6b45_grn"]];
```
Comment (jaj22) [inline A3-Antistasi/Templates/NewTemplates/RHS/RHS_AI_AFRF_Arid.sqf:260]: I feel like players are gonna be complaining about invincible russians, but whatever.

**Review by jaj22 — CHANGES_REQUESTED** [review]
Just the shotgun consistency thing really. I'll miss the random smoke colours but they were a bit silly.

File: A3-Antistasi/Templates/NewTemplates/RHS/RHS_AI_AFRF_Arid.sqf (hunk 1 above)
Comment (Lazejun) [reply A3-Antistasi/Templates/NewTemplates/RHS/RHS_AI_AFRF_Arid.sqf]: Updated All to shotguns

**Review by jaj22 — APPROVED** [review]

**Pull request merged**